src/stream/mod.rs

```diff
@@ -69,27 +70,76 @@ enum StackItem {
     DictValue(Value),
 }
 
+/// Options for customizing serialization of XML plists.
+#[derive(Clone, Debug)]
+pub struct WriteOptions {
+    pub(crate) sort_keys: bool,
+    pub(crate) indent_str: &'static str,
+    __nonexhaustive: (), // we may add more options in the future.
+}
+
 impl IntoEvents {
     pub(crate) fn new(value: Value) -> IntoEvents {
         IntoEvents {
             stack: vec![StackItem::Root(value)],
+            sort_dict_keys: false,
```

Comment:
I'd rather not plumb the sorting option all the way into `IntoEvents`. It would be better as a single function that takes an `&mut Value`.